Microsoft.HBase.Client.Tests/VirtualNetworkLoadBalancerTests.cs

```diff
@@ -29,6 +29,7 @@ namespace Microsoft.HBase.Client.Tests
     using Microsoft.VisualStudio.TestTools.UnitTesting;
     using org.apache.hadoop.hbase.rest.protobuf.generated;
 
+    /*
```

Comment:
I will fix this test file in a follow up ticket, also adding some more significant integration tests with linux and vnet clusters